osmoutils/accum/accum.go

```diff
@@ -0,0 +1,98 @@
+package accum
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/store"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/gogo/protobuf/proto"
+)
+
+// We keep this object as a way to interface with the methods, even though
+// only the Accumulator inside is stored in state
+type AccumulatorObject struct {
+	// Store where accumulator is stored
+	Store store.KVStore
```

Comment:
Out of curiosity, but why do we need to know where the accumulator is stored? Can't we just store the key and reference that key? 

aah is it cause we usually store the key in keeper but we don't have that here?

Reply:
Well the store is passed into the make and get functions since we don't have the key here yes, but we keep it in this accumulator object so that the caller doesn't have to constantly pass in their store for every function.

i.e. intended workflow: call get function with store -> call every other function on the returned object, not having to worry about passing in the correct store for each

Note that only `AccumulatorContent` is marshaled and placed into the store – `AccumulatorObject` is simply a wrapper for convenience